src/main/java/org/fundacionjala/at15/pacman/InsertBoard.java

```diff
@@ -0,0 +1,52 @@
+package org.fundacionjala.at15.pacman;
```

Comment:
This file might has been added accidentally, keep an eye on the files you add before creating your commits